apps/api/openapi/openapi.json

```diff
@@ -17,10 +17,6 @@
     {
       "url": "http://localhost:3000",
       "description": "Local development server"
-    },
-    {
-      "url": "https://api.example.com",
```

Comment:
Looks like I didn't have to remove this, but I took it out for simplicity while investigating.

Reply:
(Note: if we end up merging this PR, we should add this public URL back but change it to the competitions API URL: api.competitions.recall.network